asserts.lisp

```diff
@@ -50,6 +50,56 @@ vice versa."
   `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'equalp
     :full-form ',whole))
 
+(defmacro assert-string= (&whole whole expected form &rest extras)
+  "Assert whether expected and form are STRING=."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'string=
+    :full-form ',whole))
+
+(defmacro assert-string-equal (&whole whole expected form &rest extras)
+  "Assert whether expected and form are STRING-EQUAL."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'string-equal
+    :full-form ',whole))
+
+(defmacro assert-string/= (&whole whole expected form &rest extras)
+  "Assert whether expected and form are STRING/=."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'string/=
+    :full-form ',whole))
+
+(defmacro assert-string-not-equal (&whole whole expected form &rest extras)
+  "Assert whether expected and form are STRING-NOT-EQUAL."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'string-not-equal
+    :full-form ',whole))
+
+(defmacro assert-char= (&whole whole expected form &rest extras)
+  "Assert whether expected and form are CHAR=."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'char=
+    :full-form ',whole))
+
+(defmacro assert-char-equal (&whole whole expected form &rest extras)
+  "Assert whether expected and form are CHAR-EQUAL."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'char-equal
+    :full-form ',whole))
+
+(defmacro assert-char/= (&whole whole expected form &rest extras)
+  "Assert whether expected and form are CHAR/=."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'char/=
+    :full-form ',whole))
+
+(defmacro assert-char-not-equal (&whole whole expected form &rest extras)
+  "Assert whether expected and form are CHAR-NOT-EQUAL."
+  `(expand-assert 'equal-result ,form ,form ,expected ,extras :test #'char-not-equal
+    :full-form ',whole))
+
+(defmacro assert-= (&whole whole expected form &rest extras)
```

Comment:
Note that equal sign and `/=` sign is not preceded by the hyphen in other equality operators/asserts. Maybe remove hyphen?

Reply:
I agree. Perhaps if we'd follow the style we use at Nyxt, we'd do otherwise but...